pkg/controller/migplan/validation.go

```diff
@@ -48,6 +48,7 @@ const (
 	NsNotFoundOnDestinationCluster             = "NamespaceNotFoundOnDestinationCluster"
 	NsLimitExceeded                            = "NamespaceLimitExceeded"
 	NsLengthExceeded                           = "NamespaceLengthExceeded"
+	NsNotDNSCompliant                          = "NotDNSCompliantNamespece"
```

Comment:
Nit: s/NotDNSComplianteNamespace/NamespaceNotDNSCompliant/ to be consistent with the strings above